bin/main

```diff
@@ -1,134 +1,139 @@
 #!/usr/bin/env ruby
 
+require './lib/game_logic.rb'
+
```

Comment:
### [OPTIONAL] 
Using require_relative is in preference to using require. Kindly use require_relative to link your files